_data/contributions.yml

```diff
@@ -181,3 +181,6 @@ devs:
   - github_url: "https://github.com/umershaikh97/"
     name: "Umer Shaikh"
     avtar: "https://avatars.githubusercontent.com/umershaikh97"
+  - github_url: "https://github.com/kuroyza"
+    name: "Kuroyza"
+    avtar: "https://avatars.githubusercontent.com/kuroyza"
```

Comment:
Please fix this spell mistake, avatar instead of avtar.

Reply:
Ok I'm going to work on it, Thank you so much for your notice.